README.rst

```diff
@@ -31,6 +31,8 @@ starfish: scalable pipelines for image-based transcriptomics
 
 -------------------------------
 
+SILLY CHANGE
+
```

Comment:
omg

Reply:
lol I was demonstrating how CI works for Lucy :P

Reply:
Deep.

Reply:
you rang?

Reply:
We miss you.

Reply:
AWW DEEP